src/main/scala/beam/sim/BeamServices.scala

```diff
@@ -75,6 +76,7 @@ class BeamServicesImpl @Inject()(val injector: Injector) extends BeamServices {
   )
 
   var modeChoiceCalculatorFactory: ModeChoiceCalculatorFactory = _
+  var metricsPrinter: ActorRef = _
```

Comment:
I realize this is done elsewhere - but it's a bit dangerous. Could we at least force implementers to acknowledge the possibility of emptiness by making this an `Option` at least?

Reply:
Moved to BeamSim, printer is not needed in services.